_includes/header.html

```diff
@@ -16,24 +16,24 @@
                     <li>
                         <a href="/" title="There's no place like Home...">Home</a>
                     </li>
+                    <li>
+                        <a href="/docs/getting-started" title="Our Getting Started guide">Get Started</a>
```

Comment:
let's do the downloads page in a separate PR. there are enough changes in this one already!